tests/nsrt_learning/test_option_learning.py

```diff
@@ -227,3 +231,40 @@ def test_create_option_learner():
     })
     with pytest.raises(NotImplementedError):
         create_option_learner()
+
+
+@longrun
+def test_option_learning_approach_multistep_cover():
+    """A long test to identify any regressions in option learning."""
+    utils.reset_config({
+        "env": "cover_multistep_options",
+        "approach": "nsrt_learning",
+        "option_learner": "neural",
+        "sampler_learner": "oracle",
+        "num_train_tasks": 10,
+        "num_test_tasks": 10,
+    })
+    env = create_new_env("cover_multistep_options")
+    train_tasks = env.get_train_tasks()
+    approach = create_approach("nsrt_learning", env.predicates, env.options,
+                               env.types, env.action_space, train_tasks)
+    dataset = create_dataset(env, train_tasks)
+    assert approach.is_learning_based
+    approach.learn_from_offline_dataset(dataset)
+    num_test_successes = 0
+    for task in env.get_test_tasks():
+        try:
+            policy = approach.solve(task, timeout=CFG.timeout)
+            traj = utils.run_policy_with_simulator(policy,
+                                                   env.simulate,
+                                                   task.init,
+                                                   task.goal_holds,
+                                                   max_num_steps=CFG.horizon)
+            if task.goal_holds(traj.states[-1]):
+                num_test_successes += 1
+        except (ApproachFailure, ApproachTimeout):
+            continue
+    # This number is expected to be relatively low because the number of train
+    # tasks is pretty limiting. But if it goes lower than this, that could
+    # be a performance regression that we should investigate.
```

Comment:
feel free to make the num train tasks higher if you want. btw, this file with longrun takes 60 seconds for me locally:
`pytest -s tests/nsrt_learning/test_option_learning.py --longrun`
and gives me a num_test_successes of exactly 3. so maybe we want more train tasks or a lower threshold?

Reply:
resolved over slack